node/config/types.go

```diff
@@ -282,7 +282,8 @@ type Splitstore struct {
 	HotStoreType  string
 	MarkSetType   string
 
-	HotStoreMessageRetention uint64
+	HotStoreMessageRetention  uint64
+	HotStoreMovingGCFrequency uint64
```

Comment:
Would be nice to add docstrings to those since we now expose those comments in user configs automagically

Reply:
Ah nice, I didn't know about this. I will add a one-liner for them, as we have detailed documentation in the README.

Reply:
done.